tests/test_flask_pyoidc.py

```diff
@@ -162,19 +163,14 @@ def test_should_register_client_if_not_registered_before(self, post_logout_redir
         self.app.add_url_rule('/logout', view_func=logout_view_mock)
         authn.oidc_logout(logout_view_mock)
 
-        responses.add(responses.POST, registration_endpoint, json={'client_id': 'client1', 'client_secret': 'secret1'})
+        responses.add(responses.POST, registration_endpoint, json={
+            'client_id': 'client1', 'client_secret': 'secret1',
+            'redirect_uris': ['https://client.example.com/redirect']})
         view_mock = self.get_view_mock()
         with self.app.test_request_context('/'):
             auth_redirect = authn.oidc_auth(self.PROVIDER_NAME)(view_mock)()
 
         self.assert_auth_redirect(auth_redirect)
-        registration_request = json.loads(responses.calls[0].request.body.decode('utf-8'))
```

Comment:
Why was this removed?

Reply:
Now `client.register` method is registering the client which returns a response object. This line is no longer required.

Reply:
This is checking what the registration request body was which should still be possible since pyoidc also uses `requests` library under the hood.
It does not depend on the response directly.